src/xpk/commands/cluster.py

```diff
@@ -321,6 +321,7 @@ def cluster_describe(args) -> None:
       'Count TPU Nodes',
       args,
   )
+  node_output = node_output.splitlines()[-1]
```

Comment:
why don't we check return code before splitlines?

Reply:
Agreed. Changed.